UnitySDK/Assets/ML-Agents/Scripts/Monitor.cs

```diff
@@ -8,6 +8,7 @@ namespace MLAgents
     /// Monitor is used to display information about the Agent within the Unity
     /// scene. Use the log function to add information to your monitor.
     /// </summary>
+    [AddComponentMenu("ML Agents/Monitor")]
```

Comment:
I'm OK removing this one too.

Reply:
We should, this is supposed to be hidden from the user and automagic